chpl/chpl-resources/src/main/resources/quartz.properties

```diff
@@ -7,7 +7,7 @@ org.quartz.jobStore.tablePrefix = quartz.QRTZ_
 org.quartz.jobStore.isClustered = true
 
 org.quartz.plugin.jobInitializer.class = org.quartz.plugins.xml.XMLSchedulingDataProcessorPlugin
-org.quartz.plugin.jobInitializer.fileNames = jobs.xml,ocd-3149.xml
+org.quartz.plugin.jobInitializer.fileNames = jobs.xml,ocd-3149.xml,ocd-3122.xml
```

Comment:
When approved for merge to STG, I thought we merged the ocd-nnnn.xml files into the jobs.xml file